tests/functional/prometheous/test_prometheus_validations_file.py

```diff
@@ -333,3 +335,65 @@ def test_prometheus_pv_resize(self):
             pod_name=pod_name, pvc_name=pvc_name,
             filename="secondfilename", dirname="seconddirname",
             metric_data=resize_metrics, operation="create")
+
+    @pytest.mark.tier3
+    def test_prometheus_volume_metrics_on_node_reboot(self):
+        """Validate volume metrics using prometheus before and after node
+        reboot"""
+
+        # Create I/O pod with PVC
+        pvc_name = self.create_and_wait_for_pvc()
+        prefix = "autotest-{}".format(utils.get_random_str())
```

Comment:
Move this declaration to start of the test

Reply:
done